app/services/mentions/create_all.rb

```diff
@@ -48,29 +48,38 @@ def extract_usernames_from_mentions_in_text
     end
 
     def reject_notifiable_author(users)
-      users.reject { |user| authored_by?(user, @notifiable) }
+      users.reject { |user| authored_by?(user, notifiable) }
```

Comment:
Consolidated any instance variables of `@notifiable` to `notifiable` directly to reduce future confusion (this should be equivalent given that we have an `attr_reader :notifiable` at the bottom of the file)